_posts/2025-09-16-generative-ai-peer-review.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: single
+title: "Navigating LLMs in Open Source: pyOpenSci's New Peer Review Policy"
+excerpt: "Generative AI tools are making is easier to generate large amounts of code which in some cases is causing a strain  on volunteer peer review programs like ours. Learn about pyOpenSci's policy on generative AI in peer review in this blog post."
+author: "pyopensci"
+permalink: /blog/generative-ai-peer-review-policy.html
+header:
+  overlay_image: images/headers/pyopensci-floral.png
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+last_modified: 2025-09-16
+---
+
+authors: Leah Wasser, Mandy Moore, 
+
+## Generative AI meets scientific open source
+
+It has been suggested that for some developers, using AI tools for tasks can increase efficiency by as much as 55%. But in open source scientific software, speed isn't everything—transparency, quality, and community trust matter just as much. So do the ethical questions these tools raise.
+
+**Edit this.** Whatever breakout content we want here.... needs to be all on a single line. 
+{: .notice--success}
+
+
+## Why we need guidelines
+
+At [pyOpenSci](https://www.pyopensci.org/), we’ve drafted a new policy for our peer review process to set clear expectations around disclosing use of LLMs in scientific software packages.
+
+This is not about banning AI tools. We recognize their value to some. Instead, our goal is transparency. We want maintainers to **disclose when and how they’ve used LLMs** so editors and reviewers can fairly and efficiently evaluate submissions.
+
+## Our Approach: Transparency and Disclosure
+
+We know that people will continue to use LLMs. We also know they can meaningfully increase productivity and lower barriers to contribution for some. We also know that there are significant ethical, societal and other challenges that come with the development and use of LLM’s. 
+
+Our community’s expectation is simple: **be open about it**.
+
+* Disclose LLM use in your README and at the top of relevant modules.  
+* Describe how the tools were used   
+* Be clear about what human review you performed.
+
+Transparency helps reviewers understand context, trace decisions, and focus their time where it matters most.
+
+### Human oversight
+
+LLM-assisted code must be **reviewed, edited, and tested by humans** before submission.
+
+* Run tests and confirm correctness.  
+* Check for security and quality issues.  
+* Ensure style, readability, and clear docstrings.  
```

Comment:
Ensure style, readability, clear, and concise docstrings.

Depending on the AI tool, generated docstrings can sometimes be overly verbose without adding meaningful understanding.